src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/ImageRenderer.js

```diff
@@ -172,7 +171,6 @@ class ImageRenderer extends React.Component<Props> {
                     {this.imageUrl &&
                         <img
                             className={imageRendererStyles.image}
-                            key={this.imageUrl}
```

Comment:
Didn't that help with the rendering problems? Or did you solve it in a different way?

Reply:
When the `<img />` had the key, on every single update of a hotspot (moving, resizing, changing form, ...) a request to `medias` has been made

Reply:
fixed